apollo-runtime/src/main/java/com/apollographql/apollo/ApolloMutationCall.java

```diff
@@ -0,0 +1,41 @@
+package com.apollographql.apollo;
+
+import com.apollographql.apollo.api.Mutation;
+import com.apollographql.apollo.api.OperationName;
+import com.apollographql.apollo.cache.CacheHeaders;
+
+import javax.annotation.Nonnull;
+
+/**
+ * A call prepared to execute GraphQL mutation operation.
+ */
+public interface ApolloMutationCall<T> extends ApolloCall<T> {
+
+  /**
+   * <p>Sets a list of GraphQL query names to be re-fetched once this mutation completed.</p>
```

Comment:
I was imagining you would just pass the actual queries to be refetched here instead of an identifier. 

Also, what would happen if I was listening to the same query that was executed twice with different variables. Seems like the tags would collide.